en/manual/02-getting-started.md

```diff
@@ -168,23 +168,23 @@ and run Job definitions and interact with the dispatcher queue. These
 command tools are an alternative to functions accessible in the
 graphical console.
 
-See the [Command line tools](../man1/index.html).
+See the [Command line tools][page:manpages/man1/index.md].
 
 ## API
 
 You can also use the Web API to interface with all aspects of node
 and Job execution. 
 
-See the [Rundeck API](../api/index.html) page for a reference on the
+See the [Rundeck API][page:api/rundeck-api.md] page for a reference on the
 endpoints and examples.
 
 ## Document Formats
 
 If you prefer to manage job and resource definitions using text files
 you can do so using XML or YAML formats.
 
-See the [Document Format Reference](../man5/index.html).
+See the [Document Format Reference][page:manpages/man5/index.md].
 
-[node]: nodes.html
-[command]: commands.html
-[execution]: executions.html
+[node][page:manual/05-nodes.md]
+[command][page:manual/06-commands.md]
+[execution][page:manual/07-executions.md]
```

Comment:
More link reference definitions that were converted to actual links

Reply:
fixed...